openwisp_monitoring/device/admin.py

```diff
@@ -84,7 +106,7 @@ def __init__(self, *args, **kwargs):
 
 class AlertSettingsInline(NestedStackedInline):
     model = AlertSettings
-    extra = 0
+    extra = 1
```

Comment:
Why do we need an extra here?

Reply:
I changed `extra = 1` to get the complete form, but now it looks weird when the user has `view` permission (it shows empty input fields as well) so I reverted this change :+1: 

#### `With extra = 0`

![Screenshot from 2022-08-20 17-07-43](https://user-images.githubusercontent.com/56113566/185745825-c8e9cda8-72a4-4dd1-b251-acf81a69d309.png)

#### `With extra = 1`

![Screenshot from 2022-08-20 17-03-46](https://user-images.githubusercontent.com/56113566/185745829-cf52b10d-6ff0-425f-a313-fdcc9b01f39a.png)